ApplicationManager.js

```diff
@@ -60,7 +72,23 @@ class ApplicationManager {
       },
       {}
     );
+    return {...initializedParam}
+  }
 
+  resetParams(algoName) {
+    this.algoParams = this.get_Initialized_Params(algoName);
+  }
+
+  changeParamsAlgorithms(algoName) {
+    let new_algoParams = this.get_Initialized_Params(algoName);
+    for (let key in new_algoParams) {
```

Comment:
Same for var names convention:
```suggestion
    for (let key in newAlgoParams) {
```